src/directives/class-map.ts

```diff
@@ -21,10 +21,27 @@ export interface ClassInfo {
 /**
  * Stores the ClassInfo object applied to a given AttributePart.
  * Used to unset existing values when a new ClassInfo object is applied.
+ *
+ * The map contains two possible class values:
+ * 1. Static (given through the template literal), which are set to true
+ * 2. Dynamic (given through ClassInfo) which are set to false
+ *
+ * Only Dynamic classes can be removed by being omitted from the ClassInfo, but
+ * both Static and Dynamic classes can be removed by setting class to falsey in
+ * the ClassInfo.
  */
-const previousClassesCache = new WeakMap<Part, Map<string, unknown>>();
+const previousClassesCache = new WeakMap<Part, Map<string, boolean>>();
 
-const staticValue = {};
+/**
+ * Classes are parsed as a series of tokens separated by ASCII whitespace. This
+ * token splitter allows us to extract the tokens from the static classes given
+ * in th template literal.
```

Comment:
Typo, should be: `* in the template literal.`

Reply:
Thanks!